### PR TITLE
(fix) `Explore` Custom Sort

### PR DESCRIPTION
**Pull request by mazenchami**
# Description

[Trello Card](154-explore-custom-sort-order) — #118 
[Trello Card](170-feature-request-advanced-map-link) — #133 

- adding a custom sort function for objects by its keys
- sorting the `Explore` tab
- Advanced Map Link on `Explore` tab (feature request)

# Graphics

| iOS | Android |
| ---- | ---- |
|  | 

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

File: app/i18n/en.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ const en = {
     uniqueToPortland: "Unique to Portland",
     openInMaps: "Open in maps",
     website: "Website",
+    wantToSeeMore: "Want to see more places? Open this map!",
```
Comment (morganick) [inline app/i18n/en.ts:40]: Any way we can shorten this text? (e.g. View our destinations map)
image

**Review by morganick — COMMENTED** [review]
🙌 for the function comments!

File: app/i18n/en.ts (hunk 1 above)
Comment (mazenchami) [reply app/i18n/en.ts:40]: after convo with gant, we're going with the `reversed` preset. screenshot updated and code updated here: https://github.com/infinitered/ChainReactApp2023/pull/150/commits/ab070d2b8f5066983b57fbaa95463131787313ef

File: app/screens/ExploreScreen/ExploreScreen.tsx [hunk 2]
```diff
@@ -102,6 +105,19 @@ export const ExploreScreen: FC<TabScreenProps<"Explore">> = () => {
       )}
       {!isLoading && sections.length > 0 && (
         <SectionList
+          ListFooterComponent={
+            <ButtonLink
+              openLink={() =>
+                openLinkInBrowser(
```
Comment (morganick) [inline app/screens/ExploreScreen/ExploreScreen.tsx:111]: Opening this link doesn't appear to work on android both on device and in sim.

Comment (mazenchami) [reply app/screens/ExploreScreen/ExploreScreen.tsx]: fixed here: https://github.com/infinitered/ChainReactApp2023/pull/150/commits/abb1d403c239d30a3d99d3f2833ad46ff9bbed63

**Review by morganick — APPROVED** [review]
I can confirm that the link now works. I'll like to dig in further to see what the deal is after we get through some of the other issues.

**Pull request merged**